Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM jinaai/jina:2.0.0rc6
```

Comment:
I would recommend that we add GPU support for all encoders. 
@jakob or @sebastian could help you with that since they did it for torch and tensorflow already.

Reply:
as discussed with @nan-wang GPU is not a mandatory requirement at the moment